packages/emails/src/components/BaseEmailHtml.tsx

```diff
@@ -24,78 +25,137 @@ export const BaseEmailHtml = (props: {
   subtitle?: React.ReactNode | string;
   headerType?: BodyHeadType;
   hideLogo?: boolean;
+  locale: string;
 }) => {
+  const textDirection = getTextDirection(props.locale);
+  const isRTL = textDirection === "rtl";
```

Comment:
And here as well

Reply:
yes sir that's what I did, I wrote it that way

but this was suggested 

https://github.com/calcom/cal.com/pull/16697#pullrequestreview-2312131712

and I think it'll be better to keep everything in one place, incase this needs to be updated we might need to update it only once instead of three files. 

what do both of you think?

Reply:
Yes that's fine. I meant calling the isRTL function directly since it calls getTextDirection and matches if it is "rtl", instead of you calling getTextDirection here and then manually matching it everywhere

https://github.com/calcom/cal.com/blob/e6f1cd84a9b7f8874758e512c66ea914eb207b1a/packages/lib/i18n.ts#L23-L25

```suggestion
  const isRightToLeft = isRTL(props.locale);
```

Reply:
https://github.com/calcom/cal.com/pull/16697#pullrequestreview-2312131712  Also this comment means you should use `Intl.Locale` and fetch information about its alignment instead of us maintaining a list, would be easier in case we add more languages.